tests/core/channels/test_telegram.py

```diff
@@ -0,0 +1,66 @@
+import json
+import logging
+from unittest.mock import patch
+
+import pytest
```

Comment:
Could you please remove this unused import too as it's been flagged by the CI test?